src/statistics.py

```diff
@@ -0,0 +1,55 @@
+
+class StatisticsClass:
+    def __init__(self, expec, npop, table, thres_stat=1.0e-5):
+        self.df_expec = expec
+        self.df_npop = npop
```

Comment:
Ihno requested on my part that all file-related inputs should be passed as a single parameter, because otherwise the signature could get arbitrary long. So for example pass `data` instead of `expec` and `npop` where `data` is a dictionary. Compare my `__init__` in the `NumericsClass`.